vllm/engine/metrics.py

```diff
@@ -135,7 +141,7 @@ def __init__(self, local_interval: float, labels: Dict[str, str]) -> None:
         self.labels = labels
         self.metrics = Metrics(labelnames=list(labels.keys()))
 
-    def info(self, type: str, obj: object) -> None:
+    def info(self, type: str, obj: SupportsMetricsInfo) -> None:
```

Comment:
QQ: should we change the var name obj?

Reply:
it would slight improve readability, but I am not sure if there is any dependency on the kwarg name. `obj` is no a great variable name, I agree.